interop-tests/src/bin/wasm_ping.rs

```diff
@@ -14,7 +14,7 @@ use tokio::process::Child;
 use tokio::sync::mpsc;
 use tower_http::cors::CorsLayer;
 use tower_http::trace::TraceLayer;
-use tracing::{error, warn};
+use tracing;
```

Comment:
Shouldn't need these because that is the crate name! If you run clippy, it should actually tell you :)